content/en/real_user_monitoring/error_tracking/reactnative.md

```diff
@@ -0,0 +1,424 @@
+---
```

Comment:
Should this file be changing as part of this PR? Is it going to conflict with the un-single-sourcing of the React Native documentation?